### DISCUSSION TITLE
Login with Google [Auth]

### DISCUSSION THREAD

**Discussion by MatejaSrejic** [kreait/firebase-php#530] [Q&A]
How do I get the ID Token required?
In the Firebase Console I could only find _Web Client ID_ & _Web Client Secret_

**Comment by jeromegamez** [comment]
The ID token is a token that you can get from a currently logged-in user in a client (web/app)

See https://firebase.google.com/docs/auth/admin/verify-id-tokens#retrieve_id_tokens_on_clients
